intro_to_r/intro_to_r.Rmd

```diff
@@ -239,9 +251,21 @@ of operations.  Here, we want to divide the number of boys by the total number
 of newborns, so we have to use parentheses.  Without them, R will first do the 
 division, then the addition, giving you something that is not a proportion.
 
+We can also pipe data into plots, as in
+```{r piped-plot}
+arbuthnot %>%
```

Comment:
I'm not a fan of this either.  In fact, I would avoid using the `.` in piped commands, certainly early on.  There's enough going on already without introducing this.

And I'm not sure I would call `.` and *operator*.  What is it operating upon?